frontend/pages/hosts/ManageHostsPage/_styles.scss

```diff
@@ -178,6 +178,33 @@
       }
     }
 
+    @media (max-width: $break-sm) {
```

Comment:
I kept the `max-width`, mobile-last format in this stylesheet because reversing it was going to require too much rewriting and the existing formatting was easy enough to read from a maintenance standpoint.